src/dev/precommit_hook/casing_check_config.js

```diff
@@ -71,6 +71,8 @@ export const IGNORE_FILE_GLOBS = [
   'x-pack/plugins/apm/e2e/**/*',
 
   'x-pack/plugins/maps/server/fonts/**/*',
+  // packages for the ingest manager's api integration tests could be valid semver which has dashes
+  'x-pack/test/ingest_manager_api_integration/apis/fixtures/test_packages/**/*',
```

Comment:
@jfsiii @skh @neptunian @nchaulet this is so we can have a valid semver as the package directory name with a dash in it. Should this only cover the packages directory? Or would it be useful if it was higher up like:
`'x-pack/test/ingest_manager_api_integration/apis/fixtures/**/*` ?

Reply:
let's start here and lift it up later if we need to